### DISCUSSION TITLE
RS485 vs ScreenLogic?

### DISCUSSION THREAD

**Discussion by NathanSweet** [tagyoureit/nodejs-poolController#428] [Q&A] [answered]
Can everything done via ScreenLogic also be done via RS485?

And the reverse, can everything done via RS485 also be done via ScreenLogic?

**Comment by rstrouse** [comment, marked as answer]
Yes if you hook njsPC up to it, it will automatically detect your Easy/IntelliTouch controller.  You can gang the wires together as RS485 is a bus.

**Reply by NathanSweet** [reply]
Thanks. Is there any advantage to having both RS485 and ScreenLogic? I suppose I'd have the ScreenLogic UI for anything not covered by njsPC (if there is anything).

I'm thinking that if I do get the ScreenLogic adapter, I could communicate with it directly, eg via [parnic/node-screenlogic](https://github.com/parnic/node-screenlogic), then I don't need to setup an RPi. Would I be missing out on any important functionality?

TBH, so far all I really care about is turning the lights and spa on/off. 😃 Also for disclosure, I'm using my home grown automation software, so I don't benefit from Hubitat/etc integrations.

**Reply by rstrouse** [reply]
In a word yes.  You will find your go to interface is dshPanel. njsPC also has a socket, Rest, and MQTT api.  You can configure interface files that can talk to your homegrown HA system.

**Reply by tagyoureit** [reply]
We cover everything that is available with ScreenLogic.  Parnic did a great job but you still need a computer hooked up to take advantage of the comms and his app is only an API.  We don't talk with his app.  And with our suite of products you can also extend your system with any additional hardware or software you may want to add.  I had screenlogic and definitely would not spend the money on it.  The dashpanel ui is customizable and very much more intuitive and user friendly.

**Reply by mhmatttu** [reply]
my sister has screenlogic on their pool and it is not user friendly at all.  very little config options.  njsPC is the way to go!
